java/code/src/com/suse/manager/reactor/utils/RhelUtils.java

```diff
@@ -427,6 +442,14 @@ public static Optional<RhelProduct> detectRhelProduct(
         );
     }
 
+    private static String getBaseProductName(String majorVersion) {
+        Matcher versionMatcher = Pattern.compile("^(\\d+)").matcher(majorVersion);
+        if (versionMatcher.matches() && Integer.parseInt(versionMatcher.group(1)) >= 9) {
+            return "el-base";
+        }
+        return "rhel-base";
+    }
+
```

Comment:
AFAIK we don't support CentOS 9 Stream, but it will pass here as `el-base` if anyone ever tries to onboard it. I don't think it's really important, just for us to remember.